src/main/java/com/microsoft/cse/helium/app/models/ActorsRepository.java

```diff
@@ -6,13 +6,14 @@
 import org.springframework.stereotype.Repository;
 
 import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 
 /**
  * ActorsRepository
  */
 @Repository
 public interface ActorsRepository extends ReactiveCosmosRepository<Actor, String> {
-    Flux<Actor> findByActorId(String actorId);
-   //Flux<Actor> findByTextSearchContainingOrderByActorId(String actorName, Pageable pageable);
+    Mono<Actor> findByActorId(String actorId);
+    Flux<Actor> findByTextSearchContainingOrderByActorId(String nameString);
```

Comment:
From the C# version, the name should be GetActorsByQuery() I believe.  Let's shorten try to match C# & Typescript version as closely as possible.

Reply:
Unfortunately, if we are to use springboot facace, which we will not once we completely get rid of springboot data layer, we have to use the naming convention findBy... That is how spring figures out which queries to generate. When #92 is done and #93 is being worked on, I will remove the repository pattern completely and this will be solved thus.